vars/mavenExecute.groovy

```diff
@@ -117,8 +123,17 @@ void call(Map parameters = [:]) {
         if (defines?.trim()){
             command += " ${defines}"
         }
-        dockerExecute(script: script, dockerImage: configuration.dockerImage, dockerOptions: configuration.dockerOptions) {
-            sh command
+        if (configuration.returnStdout) {
```

Comment:
The dockerExecuteStatement alongside with the closure is the same in the `if` and in the `else` block. The only difference is: in the `if` block it is assiged to a variable, and in the `else` block not. I would like to suggest to have such kind of duplicate code. Why not assigning it always to `commandOutput`? Having such duplicated coding makes code maintenance more complicated later. Some develop coming after us might have to adjust something here. Having duplicate code makes it likely to only adjust one occurance and forget about the other.

Reply:
done